File: javascript/core-concepts/pre-recorded/pre-recorded.js
import { GladiaClient } from '@gladiaio/sdk';
import 'dotenv/config';

const gladiaClient = new GladiaClient();
const audioPath = '../data/online-meeting-example.mp4';

const result = await gladiaClient.preRecorded().transcribe(audioPath);

console.log(result.result?.transcription?.full_transcript ?? '');

File: python/core-concepts/pre-recorded/pre_recorded.py
# pip install gladiaio-sdk
from gladiaio_sdk import GladiaClient

# Create your account and get your API key in 30 seconds ! [Click here](https://docs.gladia.io/chapters/introduction/getting-started) to get started.
gladia_client = GladiaClient(api_key="GLADIA_API_KEY").prerecorded()

transcription = gladia_client.transcribe(
    audio_url="../data/anna-and-sasha-16000.wav",
    options={
        "language_config": {
            # check all the supported languages at https://docs.gladia.io/chapters/language/supported-languages#supported-languages
            "languages": ["en"],
        },
    },
)

print(transcription.result.transcription.full_transcript)
